src/teacher.rb

```diff
@@ -0,0 +1,19 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
+    super(age, name)
```

Comment:
- [ ] Kindly make sure to add `parent_permission: true` to the initialize method. 
- [ ] Kindly make sure to add `parent_permission` to the constructor super.